### PR TITLE
install.nsi tàu 字體

### PR DESCRIPTION
**Pull request by niauah**

### CODE REVIEW

**Comment by niauah** [comment]
```
Error: Can't change target charset after data already got compressed or header already changed!
Error in script "output\install.nsi" on line 10 -- aborting creation process
error building weasel...
```

檢查：
```
~/git/weasel/output$ file *.nsh
FontName.nsh: ASCII text, with CRLF line terminators
FontReg.nsh:  ASCII text
```

**Comment by niauah** [comment]
Tshun ê khang-khue:

- [x]  rime-hanlo e weasel.custom.yaml thai-tiau
- [x] kai weasel.yaml `font-face`
- [ ] ka nsis components page)
- [ ] nsis uninstall fonts)

**Comment by niauah** [comment]
Tshun ê khang-khue:
1. rime-hanlo e weasel.custom.yaml thai-tiau
2. kai weasel.yaml `font-face`
(3. ka nsis components page)
(4. nsis uninstall fonts)

File: output/FontReg.nsh [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+ 
+"${Index}-9x:"
+  StrCpy $R1 "Software\Microsoft\Windows\CurrentVersion\Fonts"
+  goto "${Index}-GO"
+ 
+"${Index}-GO:"
+  ClearErrors
+  ; !insertmacro FontName "$FONT_DIR\${FontFileName}"
+  ; pop $R2
+  StrCpy $R2 "$FONT_DIR\${FontFileName}"
```
Comment (sih4sing5hong5) [inline output/FontReg.nsh:83]: 這pîng ê程式攏巡過--ah，應該無問題，用重開看袂準 #13

File: output/install.nsi [hunk 2]
```diff
@@ -6,6 +6,11 @@
 
 Unicode true
 
+!include FontReg.nsh
+!include FontName.nsh
```
Comment (sih4sing5hong5) [inline output/install.nsi]: 無效ê檔案提掉應該就好--ah

Comment (niauah) [reply output/install.nsi]: 好

File: output/FontReg.nsh (hunk 1 above)
Comment (niauah) [reply output/FontReg.nsh:83]: 了解

**Review by sih4sing5hong5 — APPROVED** [review]
若是 新版build 有成功，就先merge

**Comment by niauah** [comment]
Build OK
https://github.com/i3thuan5/Rime-HanLo/releases/tag/untagged-e119e526e3b2ffa09120

**Pull request merged**